src/ViewUser.js

```diff
@@ -818,7 +823,10 @@ class ViewUser extends React.Component {
               handleAddRecords={this.handleAddRecords}
             />
           </Layer>
-          <Layer isOpen={query.layer ? query.layer === 'account' : false} label="Account Actions History">
+          <Layer
+            isOpen={query.layer ? query.layer === 'account' : false}
```

Comment:
I see that this logic is repeated. Let's move it the method.

`isLayerOpen('account')`